pandas/tslib.pyx

```diff
@@ -2301,52 +2323,34 @@ class Timedelta(_Timedelta):
 
         self._ensure_components()
         if self._ns:
-           return "ns"
+           return "N"
         elif self._us:
-           return "us"
+           return "U"
         elif self._ms:
-           return "ms"
+           return "L"
```

Comment:
@jreback where do these 1-letter abbreviations come from?  im trying to sort out all the many aliases we have floating around